module/zstd/zfs_zstd.c

```diff
@@ -361,6 +361,66 @@ zstd_enum_to_level(enum zio_zstd_levels level, int16_t *zstd_level)
 	return (1);
 }
 
+/*
+ * Since it was never used, it's not here, but if you need it,
+ * a copy of zfs_get_hdrversion can be found in zdb.c
+ */
+
+static size_t
+zfs_set_hdrversion(zfs_zstdhdr_t *blob, uint32_t version)
+{
+	BF32_SET(blob->raw_version_level, 0, 24, version);
+	return (0);
+}
+
+static uint8_t
+zfs_get_hdrlevel(const zfs_zstdhdr_t *blob)
+{
```

Comment:
is there a way to share this code with zdb's version?  Perhaps by putting it in a header file as an inline function?  The same could be done for the setting functions.

Reply:
Sure, done.